src/test/scala/org/scalameter/MethodInvocationCountTest.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalameter
+
+class MethodInvocationCountTest extends InvocationCountMeasurerTest {
+  test("MethodInvocationCountTest.allocations should be deterministic") {
+    checkInvocationCountMeasurerTest(new org.scalameter.examples.MethodInvocationCountTest1)
```

Comment:
This seems to work OK (check only calling manually `finalize` after a few `executeTests` because JVM ignored `System.gc`)
```
override def finalize(): Unit = {
  measurer.afterExecution(currentContext)
  super.finalize(BasePerformanceTest.setupzipper.value.result.context)
}
```